### DISCUSSION TITLE
Welcome to Discussions! / QQ 群 / Telegram Group

### DISCUSSION THREAD

**Discussion by Mister-Hope** [vuepress-theme-hope/vuepress-theme-hope#531] [Annoucement]
## Intro / 介绍

You are free to open any question about VuePress Theme Hope and related plugins here.

您可以在此处自由提出有关 VuePress Theme Hope 和相关插件的任何问题。

---

## Group / 群组

You can join our [telegram group](https://t.me/vuepressthemehope).

中国用户可以 [加入QQ群](https://jq.qq.com/?_wv=1027&k=rATJyxGK)

<div style="display: flex">
  <img src="https://user-images.githubusercontent.com/33315834/156488212-4f9f57ef-e072-4cad-bc10-fe9392fe03c7.jpg" width="300" />
  <img src="https://user-images.githubusercontent.com/33315834/106742221-b6344980-6657-11eb-9baa-4648bf9e6b3d.png" width="300" />
</div>

**Comment by huntmarcus12** [comment]
hopeforlife
